schema/resource.go

```diff
@@ -101,6 +101,18 @@ func (r *Resource) Columns() []string {
 	return r.Table.Columns.Names()
 }
 
+// Validates various constraints. For example: Does all primary keys has values.
+func (r *Resource) Validate() error {
+	for i, c := range r.Table.Columns {
```

Comment:
How about we don't return early and report in case multiple primary keys are missing instead of only the first one?

Reply:
Good idea. Added.